src/Core/Impl/IO/PathUtils.cs

```diff
@@ -351,6 +351,11 @@ public static bool PathStartsWith(string s, string prefix)
         /// </summary>
         public static bool IsZipFile(string rawZipPath, out string zipPath) {
             var path = NormalizePathAndTrim(rawZipPath);
+            if (!File.Exists(path)) {
```

Comment:
Unfortunate thing here is that previously, this was a pure function of its inputs, but now it makes a FS access. Does this need to be something in terms of IFileSystem?

Reply:
Got it, I can change it so it takes in an IFileSystem and in the tests just make a mock one.

Reply:
I like the test you've written already, so I wouldn't worry about changing it.

I would check out FindFile in this same class; it's probably what you should be modelling.

Reply:
I mean I'll have to mock IFileSystem in PathUtilsTest so that it returns true for fs.FileExists, otherwise the unit tests will fail.

Reply:
Yes, so I can either make a new class in PathUtilsTests that implements IFileSystem and just makes FileExists return true. This way is somewhat ugly, but it's contained in one class so I don't think it's too bad. Normally, I would use a mocking library but we don't seem to have one. The alternative is just removing that test file completely and relying on the tests in ImportTests, the ones in ImportTests cover the behavior pretty well.

Reply:
We definitely have a mocking library, NSubstitute. You should be able to do what you want with it very easily.

Reply:
Great! Thanks!